pkg/scalers/redis_streams_scaler.go

```diff
@@ -166,6 +170,22 @@ func createEntriesCountFn(client redis.Cmdable, meta *redisStreamsMetadata) (ent
 			}
 			return entriesLength, nil
 		}
+	case lagFactor:
+		entriesCountFn = func(ctx context.Context) (int64, error) {
+			groups, err := client.XInfoGroups(ctx, meta.streamName).Result()
+			if err != nil {
+				return -1, err
+			}
+			numGroups := len(groups)
+			for i := 0; i < numGroups; i++ {
+				group := groups[i]
+				if group.Name == meta.consumerGroupName {
+					return group.Lag, nil
+				}
+			}
+			err = fmt.Errorf("Stream name does not exist.")
+			return int64(0), err
```

Comment:
Shouldn't we send -1 as done in the previous lines: https://github.com/mikelam-us/keda/pull/1/files#diff-15f494355bde4b066fb2b2b23d280ece433d8c67bfc535c20d8df02cbe56f2b2R169